design-proposals/eim-standalone-managed-en.md

```diff
@@ -0,0 +1,236 @@
+    Note that the SEN will only require a subset of current cloud-init that is generated for remote provisioning.
+13. Once OS profile is found and matched, all required HW/OS info provided and cloud-init generated, the OM creates Host and Instance resources.
+    The statuses should be set to Onboarded and Provisioned. Host's desired and current state should be set to `ONBOARDED`.
+    The Instance desired state should be set to `RUNNING`, while the current state to `UNSPECIFIED` (until BM agents are up and running).
+14. Once the onboarding is completed, the CLI tool receives the generated cloud-init configuration.
+15. The CLI tool saves the cloud-init under the standard path that is used by EIM.
+    The user can either run the cloud-init manually or reboot the system to trigger the cloud-init.
+16. The cloud-init would provide all necessary configs for BM agents to start. Once they boot, they should start
+    communicating with the orchestrator and the EN status should be changed to Running.
+
+## Rationale
+
+### Considerations on how to trigger SEN onboarding
+
+The design choice is to use a simple CLI that will be invoked by local on-site admin to trigger SEN onboarding.
+We discussed the following alternatives:
+
+1. **SEN onboarding triggered by injecting a USB stick with a configuration file.**
+2. **SEN onboarding triggered by copying a configuration file under a well-defined OS path**
+
+Both approaches require having a local daemon scanning for USB sticks or on-disk config files. A local daemon is more
+"heavy" solution as it would require special security permissions and a more complex lifecycle management via systemd.
+It is also less interactive, so that possible intermediate errors may be harder to debug (on the contrary the CLI tool guides users through process).
+
+Both options are still valid for future releases, but for now we decided to go with the CLI tool as the MVP solution.
+
+### Considerations on the user workflow
+
+There are two major workflows we support - bottom-up (IO) and top-down (NIO, requires EN pre-registration).
+
+In this ADR we selected to use the bottom-up approach as it requires less manual steps - user logs in to EN and
+run the entire workflow from the EN itself, without the need to access the Edge Orchestrator UI/API.
+
+Also, the IO flow doesn't require any modifications to UI.
+
+### Considerations on scaling the SEN onboarding
+
+The current design assumes that the user will SSH into a Standalone EN and trigger onboarding one by one (it can still be automated by a script).
+Depending on the customers' requirements we can provide a kind of a "Bulk Onboarding Tool" that will allow to onboard multiple ENs at once.
+
+### How to map local OS users to Local Accounts?
+
+For now, we won't import existing OS users as local accounts. If we support NIO in the future,
+we can add support for defining local accounts that should be configured on the pre-provisioned SEN that is onboarded.
```

Comment:
you can create the default-local account though if it is configured for the project